hog.py

```diff
@@ -248,8 +251,17 @@ def bacon_strategy(score, opponent_score):
     >>> bacon_strategy(50, 70)
     0
     """
-    "*** YOUR CODE HERE ***"
-    return 5 # Replace this statement
+
+    if (opponent_score < 7) or (opponent_score == 0):
+        return BASELINE_NUM_ROLLS
+    elif (opponent_score > 7) and (opponent_score < 10):
```

Comment:
The value of BACON_MARGIN - 1 is not accounted for in this elif statement or in the if statement above it. This value needs to be accounted for somewhere but the correct spot will change after you replace "7" with the BACON_MARGIN global variable.

Reply:
Likewise, the magic number "10" is not accounted for when the opponent's score is this value. Again, the quick fix is using _gte_ or _lte_ but is better to redesign this function with the suggestions in the comment below.

Additionally, you will notice that running the experiments for the bacon_strategy (by setting line 227 to True and calling the file with `python hog.py -r`) will result in an AssertionError because of the occasions when the opponent's score is either 7 or 10.